app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt

```diff
@@ -518,14 +519,16 @@ class PasswordStore : AppCompatActivity() {
         startActivityForResult(decryptIntent, REQUEST_CODE_DECRYPT_AND_VERIFY)
     }
 
+    // TODO(msfjarvis): This is hard to support with the current setup, should we retire it?
```

Comment:
Passing the password in via the suggested password mechanism seems fine to me by itself, we should just rename it to something more generic. But given that we have plenty of UI state that depends on whether this is creation/editing, a separate activity would indeed be cleaner.

What is "this" referring to in "thus make it possible to support this again?"

Reply:
Just being able to support the edit shortcut in the contextual action bar. I've never used it and only now saw it exists so I'm not too sure. The problem here would be that we can't provide a password to the activity from here because the password file hasn't been decrypted yet.